### ISSUE TITLE
Validation loss vs training loss

### ISSUE THREAD

**Issue by alfonso2166** [ACEsuit/mace#707]
Hi all,
My training goes flawlessly well. However, I was wondering if it makes sense to plot validation/training loss as a function of epochs. I did that with the results of my training and got almost identical results for both loss functions. I varied max_L and r_max, screenshot attached.
[Mace_Training.pdf](https://github.com/user-attachments/files/17849222/Mace_Training.pdf)

Specs of my training are below:

model: "MACE"
max_L: 0
r_max: 4.0
name: "mace01"
model_dir: "MACE_models"
log_dir: "MACE_models"
checkpoints_dir: "MACE_models"
results_dir: "MACE_models"
compute_stress: True 
compute_forces: True 
train_file: "train.xyz"
valid_fraction: 0.10
test_file: "test.xyz"
num_interactions: 2
correlation: 3
num_channels: 64
energy_key: "energy_xtb"
forces_key: "forces_xtb"
stress_key: "REF_stress"
error_table: "PerAtomRMSEstressvirials"
loss: "weighted" 
forces_weight: 1.0
energy_weight: 1.0
stress_weight: 1.0
config_type_weights: '{"Default":1.0}'
device: cuda
batch_size: 10
max_num_epochs: 100
swa: False
seed: 123

All the best,
Alfonso
